nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -840,7 +887,8 @@ private FieldInitEntities collectEntities(ClassTree tree, VisitorState state) {
     Set<BlockTree> instanceInitializerBlocks = new LinkedHashSet<>();
     Set<BlockTree> staticInitializerBlocks = new LinkedHashSet<>();
     Set<MethodTree> constructors = new LinkedHashSet<>();
-    Set<MethodTree> initializerMethods = new LinkedHashSet<>();
+    Set<MethodTree> instanceInitializerMethods = new LinkedHashSet<>();
+    Set<MethodTree> staticInitializerMethods = new LinkedHashSet<>();
```

Comment:
What is a static initializer method?  I think I intended the `staticInitializerBlocks` field to hold this info but never got around to implementing

Reply:
I am using both. staticInitializerBlocks for things like:

```
static {
    // Code
}
```

And staticInitializerMethods for:

```
@Initializer
public static init() {
    // Code
}

Reply:
Ah, ok, cool.  Maybe add a comment on what they are?

Reply:
Done :)